effectiveOfficeBackend/src/main/kotlin/office/effective/common/exception/WorkspaceUnavailableException.kt

```diff
@@ -0,0 +1,6 @@
+package office.effective.common.exception
+
+import java.lang.RuntimeException
+
+class WorkspaceUnavailableException(message: String): RuntimeException(message) {
+}
```

Comment:
If your class contains nothing but a constructor, there's no point in writing {}